### DISCUSSION TITLE
teslamate config web page won't open on port 4000

### DISCUSSION THREAD

**Discussion by cepheee** [teslamate-org/teslamate#2336] [Q&A]
Hello,
After installing teslamate on a raspberry pi, I can’t open the configuration web page: http://ip:4000....I get -> Unable to connect.

Here is  what i verified and what I see the logs:

docker-compose version

docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.7.3
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019

********************************************************
uname -a

Linux raspberrypi 5.10.63-v7+ #1459 SMP Wed Oct 6 16:41:10 BST 2021 armv7l GNU/Linux
********************************************************
docker-compose ps

     Name                   Command                 State                       Ports
-------------------------------------------------------------------------------------------------------
pi_database_1    docker-entrypoint.sh postgres    Up           5432/tcp
pi_grafana_1     /run.sh                          Up           0.0.0.0:3000->3000/tcp,:::3000->3000/tcp
pi_mosquitto_1   /docker-entrypoint.sh mosq ...   Up           0.0.0.0:1883->1883/tcp,:::1883->1883/tcp
pi_teslamate_1   tini -- /bin/sh /entrypoin ...   Restarting

Note:  pi_teslamate_1 process state stay in restarting mode ???
*******************************************************************
 docker-compose logs teslamate

Attaching to pi_teslamate_1
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | sleep: cannot read realtime clock: Operation not permitted
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | sleep: cannot read realtime clock: Operation not permitted
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | sleep: cannot read realtime clock: Operation not permitted
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | sleep: cannot read realtime clock: Operation not permitted
teslamate_1  | bin/teslamate: 1: /opt/app/releases/1.25.2/env.sh: Syntax error: ")" unexpected
teslamate_1  | bin/teslamate: 1: /opt/app/releases/1.25.2/env.sh: Syntax error: ")" unexpected

Note: the directory "/opt/app/releases/1.25" does not exist ??

Does anyone have an idea or advice to solve the issue?

**Comment by ktsenvolt** [comment]
https://github.com/adriankumpf/teslamate/issues/2300

Try downgrade to 1.25.0

**Reply by cepheee** [reply]
You mean uninstall and redo the installation or there is a way to downgrade easily ?

**Reply by ktsenvolt** [reply]
Guide for those who doesn't know how to downgrade:

    Log in to your Linux-box
    Shut down TeslaMate with the command: docker-compose down
    Edit the file docker-compose.yml with the command: nano docker-compose.yml
    Replace the string that states: image: teslamate/teslamate:latest with image: teslamate/teslamate:1.25.0
    How it should look before:

change-from

    How it should look after:

change-to

    Save the file by pressing Ctrl+X and pressing Y.

save

    Confirm the filename by pressing ENTER

overwrite

    To download the previous 1.25.0 version, use this command: docker-compose pull

pull

    When finished, you can start TeslaMate with the command: docker-compose up -d

That's it!

**Reply by cepheee** [reply]
Thanks for the advices that I followed.
Now It seems the teslamate process stay up but I am still getting some error (see below the log) and can not open the teslamate config web page; certainly caused by the error visible in the log.
I have not yet follow the instruction mentioned in the log, I wanted you opinion first.

pi@raspberrypi:~ $ docker-compose ps
     Name                   Command               State                    Ports
--------------------------------------------------------------------------------------------------
pi_database_1    docker-entrypoint.sh postgres    Up      5432/tcp
pi_grafana_1     /run.sh                          Up      0.0.0.0:3000->3000/tcp,:::3000->3000/tcp
pi_mosquitto_1   /docker-entrypoint.sh mosq ...   Up      0.0.0.0:1883->1883/tcp,:::1883->1883/tcp
pi_teslamate_1   tini -- /bin/sh /entrypoin ...   Up      0.0.0.0:4000->4000/tcp,:::4000->4000/tcp

******************************************************

pi@raspberrypi:~ $ docker-compose logs teslamate
Attaching to pi_teslamate_1
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | 2022-01-19 09:42:48.551 [error] Postgrex.Protocol (#PID<0.165.0>) failed to connect: ** (Postgrex.Error) ERROR 42601 (syntax_error) syntax error at or near "oid"
teslamate_1  | 2022-01-19 09:42:48.551 [error] Postgrex.Protocol (#PID<0.166.0>) failed to connect: ** (Postgrex.Error) ERROR 42601 (syntax_error) syntax error at or near "oid"
teslamate_1  | 2022-01-19 09:42:49.474 [error] Could not create schema migrations table. This error usually happens due to the following:
teslamate_1  |
teslamate_1  |   * The database does not exist
teslamate_1  |   * The "schema_migrations" table, which Ecto uses for managing
teslamate_1  |     migrations, was defined by another library
teslamate_1  |   * There is a deadlock while migrating (such as using concurrent
teslamate_1  |     indexes with a migration_lock)
teslamate_1  |
teslamate_1  | To fix the first issue, run "mix ecto.create".
teslamate_1  |
teslamate_1  | To address the second, you can run "mix ecto.drop" followed by
teslamate_1  | "mix ecto.create". Alternatively you may configure Ecto to use
teslamate_1  | another table and/or repository for managing migrations:
teslamate_1  |
teslamate_1  |     config :teslamate, TeslaMate.Repo,
teslamate_1  |       migration_source: "some_other_table_for_schema_migrations",
teslamate_1  |       migration_repo: AnotherRepoForSchemaMigrations
teslamate_1  |
teslamate_1  | The full error report is shown below.
teslamate_1  |
teslamate_1  | ** (DBConnection.ConnectionError) connection not available and request was dropped from queue after 2932ms. This means requests are coming in and your connection pool cannot serve them fast enough. You can address this by:
teslamate_1  |
teslamate_1  |   1. Ensuring your database is available and that you can connect to it
teslamate_1  |   2. Tracking down slow queries and making sure they are running fast enough
teslamate_1  |   3. Increasing the pool_size (although this increases resource consumption)
teslamate_1  |   4. Allowing requests to wait longer by increasing :queue_target and :queue_interval
teslamate_1  |
teslamate_1  | See DBConnection.start_link/2 for more information
teslamate_1  |
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/adapters/sql.ex:760: Ecto.Adapters.SQL.raise_sql_call_error/1
teslamate_1  |     (elixir 1.12.3) lib/enum.ex:1582: Enum."-map/2-lists^map/1-0-"/2
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/adapters/sql.ex:852: Ecto.Adapters.SQL.execute_ddl/4
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migrator.ex:678: Ecto.Migrator.verbose_schema_migration/3
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migrator.ex:504: Ecto.Migrator.lock_for_migrations/4
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migrator.ex:419: Ecto.Migrator.run/4
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migrator.ex:146: Ecto.Migrator.with_repo/3
teslamate_1  |     (teslamate 1.25.0) lib/teslamate/release.ex:9: anonymous fn/2 in TeslaMate.Release.migrate/0
teslamate_1  | 2022-01-19 09:43:16.664 [error] Postgrex.Protocol (#PID<0.166.0>) failed to connect: ** (Postgrex.Error) ERROR 42601 (syntax_error) syntax error at or near "oid"
teslamate_1  | 2022-01-19 09:43:16.664 [error] Postgrex.Protocol (#PID<0.165.0>) failed to connect: ** (Postgrex.Error) ERROR 42601 (syntax_error) syntax error at or near "oid"
teslamate_1  | 2022-01-19 09:43:18.776 [error] Postgrex.Protocol (#PID<0.165.0>) failed to connect: ** (Postgrex.Error) ERROR 42601 (syntax_error) syntax error at or near "oid"
teslamate_1  | 2022-01-19 09:43:19.459 [error] Could not create schema migrations table. This error usually happens due to the following:

**Reply by cepheee** [reply]
In addition, see below what I did  and what I am getting -> teslamate config page not accessible but error no more visible

- delete the volumes and recreate them:

pi@raspberrypi:~ $ docker-compose down -v
Removing network pi_default
WARNING: Network pi_default not found.
Removing volume pi_teslamate-db
Removing volume pi_teslamate-grafana-data
Removing volume pi_mosquitto-conf
Removing volume pi_mosquitto-data
****************************************
pi@raspberrypi:~ $ docker-compose up -d
Creating network "pi_default" with the default driver
Creating volume "pi_teslamate-db" with default driver
Creating volume "pi_teslamate-grafana-data" with default driver
Creating volume "pi_mosquitto-conf" with default driver
Creating volume "pi_mosquitto-data" with default driver
Creating pi_mosquitto_1 ... done
Creating pi_teslamate_1 ... done
Creating pi_database_1  ... done
Creating pi_grafana_1   ... done
************************************
pi@raspberrypi:~ $ docker-compose logs teslamate
Attaching to pi_teslamate_1
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432

**Reply by cepheee** [reply]
In order to progress in the resolution I decided to restart the raspberry; unfortunately I am facing other issue that I have not been able to solve yet: see below the sequences of the commands I executed and the result...any advice are welcome to solve the folowing ERROR: ERROR: layer does not exist

Commands:
pi@raspberrypi:~ $ sudo shutdown -r now
login as: pi
pi@192.'s password:
Linux raspberrypi 5.10.63-v7+ #1459 SMP Wed Oct 6 16:41:10 BST 2021 armv7l

The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
Last login: Wed Jan 19 10:11:50 2022 from 192.

Wi-Fi is currently blocked by rfkill.
Use raspi-config to set the country before use.

pi@raspberrypi:~ $ docker-compose up -d
Creating network "pi_default" with the default driver
ERROR: layer does not exist

************************************************************
pi@raspberrypi:~ $ docker-compose down -v
Removing network pi_default
Removing volume pi_teslamate-db
Removing volume pi_teslamate-grafana-data
Removing volume pi_mosquitto-conf
Removing volume pi_mosquitto-data

************************************************************
pi@raspberrypi:~ $ docker-compose up -d
Creating network "pi_default" with the default driver
Creating volume "pi_teslamate-db" with default driver
Creating volume "pi_teslamate-grafana-data" with default driver
Creating volume "pi_mosquitto-conf" with default driver
Creating volume "pi_mosquitto-data" with default driver
ERROR: layer does not exist

************************************************************
pi@raspberrypi:~ $ docker-compose logs teslamate
Attaching to
************************************************************
pi@raspberrypi:~ $ sudo docker-compose up -d
ERROR: layer does not exist
************************************************************
pi@raspberrypi:~ $  docker-compose up -d
ERROR: layer does not exist
************************************************************
pi@raspberrypi:~ $ docker run hello-world

Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (arm32v7)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/

************************************************************
pi@raspberrypi:~ $ sudo systemctl enable docker.service
Synchronizing state of docker.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable docker

************************************************************
pi@raspberrypi:~ $ sudo systemctl enable containerd.service

************************************************************
pi@raspberrypi:~ $ docker-compose ps
Name   Command   State   Ports
------------------------------

************************************************************
pi@raspberrypi:~ $ docker-compose restart
ERROR: No containers to restart
ERROR: 1

************************************************************
pi@raspberrypi:~ $ docker-compose up -d
ERROR: layer does not exist

************************************************************
pi@raspberrypi:~ $ docker-compose start
Starting teslamate ... failed
Starting database  ... failed
Starting grafana   ... failed
Starting mosquitto ... failed
ERROR: No containers to start
ERROR: 1

************************************************************
pi@raspberrypi:~ $ docker-compose up -d
ERROR: layer does not exist

************************************************************
pi@raspberrypi:~ $ docker-compose build
teslamate uses an image, skipping
database uses an image, skipping
grafana uses an image, skipping
mosquitto uses an image, skipping

************************************************************

pi@raspberrypi:~ $ docker-compose start
Starting teslamate ... failed
Starting database  ... failed
Starting grafana   ... failed
Starting mosquitto ... failed
ERROR: No containers to start
ERROR: 1

************************************************************
pi@raspberrypi:~ $ service docker restart
==== AUTHENTICATING FOR org.freedesktop.systemd1.manage-units ===
Authentification requise pour redémarrer « docker.service ».
Authenticating as: ,,, (pi)
Password:
==== AUTHENTICATION COMPLETE ===
************************************************************

pi@raspberrypi:~ $ docker images
REPOSITORY            TAG       IMAGE ID       CREATED        SIZE
teslamate/teslamate   latest    5a27b11b3d3a   7 days ago     122MB
teslamate/grafana     latest    ca158bc3f825   7 days ago     176MB
postgres              13        0fe9408b62ed   2 weeks ago    308MB
eclipse-mosquitto     2         b24dbb9c5469   2 months ago   9.93MB
hello-world           latest    1ec996c686eb   3 months ago   4.85kB

************************************************************
pi@raspberrypi:~ $ docker-compose up -d
ERROR: layer does not exist

************************************************************
pi@raspberrypi:~ $ docker-compose down -v
Removing network pi_default
Removing volume pi_teslamate-db
Removing volume pi_teslamate-grafana-data
Removing volume pi_mosquitto-conf
Removing volume pi_mosquitto-data

************************************************************
pi@raspberrypi:~ $ docker-compose up -d
Creating network "pi_default" with the default driver
Creating volume "pi_teslamate-db" with default driver
Creating volume "pi_teslamate-grafana-data" with default driver
Creating volume "pi_mosquitto-conf" with default driver
Creating volume "pi_mosquitto-data" with default driver
ERROR: layer does not exist

************************************************************
pi@raspberrypi:~ $ docker-compose ps
Name   Command   State   Ports
------------------------------

************************************************************
pi@raspberrypi:~ $ docker-compose logs teslamate
Attaching to

************************************************************
pi@raspberrypi:~ $  docker-compose logs
Attaching to

End

**Comment by ktsenvolt** [comment]
Guide for those who doesn't know how to downgrade:
   
   - Log in to your Linux-box
   - Shut down TeslaMate with the command: docker-compose down
   - Edit the file docker-compose.yml with the command: nano docker-compose.yml
   - Replace the string that states: image: teslamate/teslamate:latest with image: teslamate/teslamate:1.25.0
   - How it should look before:


   
   - How it should look after:


   
   - Save the file by pressing Ctrl+X and pressing Y.


   
   - Confirm the filename by pressing ENTER


   
   - To download the previous 1.25.0 version, use this command: docker-compose pull


   
   - When finished, you can start TeslaMate with the command: docker-compose up -d
That's it!
Ken Tsen  

    On Tuesday, January 18, 2022, 02:32:37 PM EST, cepheee ***@***.***> wrote:  
 
 


You mean uninstall and redo the installation or there is a way to downgrade easily ?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by refrain** [comment]
Another option instead of downgrading Teslamate is to upgrade from Debian Buster (v10) to Bullseye (v11).  You would only have to edit your /etc/apt/sources.list & /etc/apt/sources.list.d/raspi.list and change "buster" to "bullseye", then run your updates/upgrades.

**Reply by cepheee** [reply]
Ok, thanks, I’ll keep that option in mind.

**Reply by cepheee** [reply]
Finlay I did a fresh install, new os (Bullseye v11), re-installed everything; unfortunately, I am facing a different issue (see the extract of the database log  below (after the extract of the teslamate log) for more details :

Note: In the teslamate log I can see that table, indexes, trigger have been created, but suddenly the following error occurs 

All advice are welcome !!

The error from the teslamate log:
.
.
.
extract: teslamate_1  | 2022-01-21 09:26:17.564 [info] alter table charging_processes
teslamate_1  | 2022-01-21 09:26:17.567 [info] == Migrated 20190913175011 in 0.0s
teslamate_1  | 2022-01-21 09:26:17.577 [info] == Running 20190913175543 TeslaMate.Repo.Migrations.SetStartAndEndRatedRangeKm.up/0 forward
teslamate_1  | 2022-01-21 09:26:17.628 [info] == Migrated 20190913175543 in 0.0s
teslamate_1  | 2022-01-21 09:26:17.638 [info] == Running 20190925152807 TeslaMate.Repo.Migrations.CreateGeoExtensions.change/0 forward
teslamate_1  | 2022-01-21 09:26:17.639 [info] execute "CREATE EXTENSION IF NOT EXISTS cube"
teslamate_1  | ** (Postgrex.Error) ERROR 58P01 (undefined_file) could not load library "/usr/lib/postgresql/13/lib/cube.so": /usr/lib/postgresql/13/lib/cube.so: invalid ELF header
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/adapters/sql.ex:760: Ecto.Adapters.SQL.raise_sql_call_error/1
teslamate_1  |     (elixir 1.12.3) lib/enum.ex:1582: Enum."-map/2-lists^map/1-0-"/2
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/adapters/sql.ex:852: Ecto.Adapters.SQL.execute_ddl/4
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migration/runner.ex:343: Ecto.Migration.Runner.log_and_execute_ddl/3
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migration/runner.ex:117: anonymous fn/6 in Ecto.Migration.Runner.flush/0
teslamate_1  |     (elixir 1.12.3) lib/enum.ex:2385: Enum."-reduce/3-lists^foldl/2-0-"/3
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migration/runner.ex:116: Ecto.Migration.Runner.flush/0
teslamate_1  |     (ecto_sql 3.7.1) lib/ecto/migration/runner.ex:280: Ecto.Migration.Runner.perform_operation/3
teslamate_1  | 2022-01-21 09:26:50.894 [info] == Running 20190925152807 TeslaMate.Repo.Migrations.CreateGeoExtensions.change/0 forward
teslamate_1  | 2022-01-21 09:26:50.909 [info] execute "CREATE EXTENSION IF NOT EXISTS cube"
teslamate_1  | ** (Postgrex.Error) ERROR 58P01 (undefined_file) could not load library "/usr/lib/postgresql/13/lib/cube.so": /usr/lib/postgresql/13/lib/cube.so: invalid ELF header
.
.
.
*************************************************************************************

Config:

**pi@raspberrypi:~ $ cat /etc/os-release**
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

**pi@raspberrypi:~ $ docker-compose version**
docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.9.2
OpenSSL version: OpenSSL 1.1.1k  25 Mar 2021

The error from the database log:

**pi@raspberrypi:~ $ docker-compose logs database**
Attaching to pi_database_1
database_1   | The files belonging to this database system will be owned by user "postgres".
database_1   | This user must also own the server process.
database_1   |
database_1   | The database cluster will be initialized with locale "en_US.utf8".
database_1   | The default database encoding has accordingly been set to "UTF8".
database_1   | The default text search configuration will be set to "english".
database_1   |
database_1   | Data page checksums are disabled.
database_1   |
database_1   | fixing permissions on existing directory /var/lib/postgresql/data ... ok
database_1   | creating subdirectories ... ok
database_1   | selecting dynamic shared memory implementation ... posix
database_1   | selecting default max_connections ... 100
database_1   | selecting default shared_buffers ... 128MB
database_1   | selecting default time zone ... Etc/UTC
database_1   | creating configuration files ... ok
database_1   | running bootstrap script ... ok
database_1   | performing post-bootstrap initialization ... ok
database_1   | syncing data to disk ... ok
database_1   |
database_1   |
database_1   | Success. You can now start the database server using:
database_1   |
database_1   |     pg_ctl -D /var/lib/postgresql/data -l logfile start
database_1   |
database_1   | initdb: warning: enabling "trust" authentication for local connections
database_1   | You can change this by editing pg_hba.conf or using the option -A, or
database_1   | --auth-local and --auth-host, the next time you run initdb.
database_1   | waiting for server to start......2022-01-21 09:23:20.643 UTC [48] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on arm-unknown-linux-gnueabihf, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 32-bit
database_1   | .2022-01-21 09:23:22.025 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
database_1   | ..2022-01-21 09:23:23.594 UTC [49] LOG:  database system was shut down at 2022-01-21 09:22:43 UTC
database_1   | 2022-01-21 09:23:23.650 UTC [48] LOG:  database system is ready to accept connections
database_1   |  done
database_1   | server started
database_1   | CREATE DATABASE
database_1   |
database_1   |
database_1   | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
database_1   |
database_1   | waiting for server to shut down....2022-01-21 09:23:50.604 UTC [48] LOG:  received fast shutdown request
database_1   | 2022-01-21 09:23:50.654 UTC [48] LOG:  aborting any active transactions
database_1   | 2022-01-21 09:23:50.666 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
database_1   | 2022-01-21 09:23:50.667 UTC [50] LOG:  shutting down
database_1   | .2022-01-21 09:23:52.428 UTC [48] LOG:  database system is shut down
database_1   |  done
database_1   | server stopped
database_1   |
database_1   | PostgreSQL init process complete; ready for start up.
database_1   |
database_1   | 2022-01-21 09:23:52.732 UTC [1] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on arm-unknown-linux-gnueabihf, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 32-bit
database_1   | 2022-01-21 09:23:52.732 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
database_1   | 2022-01-21 09:23:52.733 UTC [1] LOG:  listening on IPv6 address "::", port 5432
database_1   | 2022-01-21 09:23:52.793 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
database_1   | 2022-01-21 09:23:52.881 UTC [62] LOG:  database system was shut down at 2022-01-21 09:23:52 UTC
database_1   | 2022-01-21 09:23:52.957 UTC [1] LOG:  database system is ready to accept connections
database_1   | 2022-01-21 09:26:17.650 UTC [75] ERROR:  could not load library "/usr/lib/postgresql/13/lib/cube.so": /usr/lib/postgresql/13/lib/cube.so: invalid ELF header
database_1   | 2022-01-21 09:26:17.650 UTC [75] STATEMENT:  CREATE EXTENSION IF NOT EXISTS cube
database_1   | 2022-01-21 09:26:50.920 UTC [79] ERROR:  could not load library "/usr/lib/postgresql/13/lib/cube.so": /usr/lib/postgresql/13/lib/cube.so: invalid ELF header
database_1   | 2022-01-21 09:26:50.920 UTC [79] STATEMENT:  CREATE EXTENSION IF NOT EXISTS cube
database_1   | 2022-01-21 09:27:25.314 UTC [83] ERROR:  could not load library "/usr/lib/postgresql/13/lib/cube.so": /usr/lib/postgresql/13/lib/cube.so: invalid ELF header
